README.md

```diff
@@ -31,3 +37,7 @@ GIT_USER=<Your GitHub username> USE_SSH=true yarn deploy
 ```
 
 If you are using GitHub pages for hosting, this command is a convenient way to build the website and push to the `gh-pages` branch.
+
+## Making Updates 
+
+Change the `SimplyPluralApi.yml` and `SimplyPluralApiBodies.yml` files instead, then run the [generator](https://github.com/PaloAltoNetworks/docusaurus-openapi-docs?tab=readme-ov-file#generating-openapi-docs)
```

Comment:
It's correct that you need to update the .yml files, but making updates shouldn't need the generating of docs. We generate the docs prior to updating it on the website, generating the docs as part of a PR will only introduce merge conflicts that can be avoided by just editing the .yml and docs files. PRs should only contain non-generated files.